src/mqtt_pal.c

```diff
@@ -380,7 +390,9 @@ ssize_t mqtt_pal_recvall(mqtt_pal_socket_handle fd, void* buf, size_t bufsz, int
 
 #else
 
-#error No PAL!
+#error No TCP PAL!
```

Comment:
i guess this is not an error.

Reply:
This `#else` block is only reached only iif the `MQTT_PAL_ENABLE_TCP` is defined but no implementation is available (neither "unix" nor "windows" one), so in that case it should fail with error. This diverges from the original case when the error was emitted in general, hence the 'No _TCP_ PAL!' message (despite being almost the same line position). Does that makes sense or I am not understanding your point here?

Reply:
ok. i have misread it.